zetaclient/app_context/app_context.go

```diff
@@ -0,0 +1,49 @@
+package appcontext
+
+import (
+	"github.com/rs/zerolog"
+	"github.com/zeta-chain/zetacore/common"
+	"github.com/zeta-chain/zetacore/zetaclient/config"
+	corecontext "github.com/zeta-chain/zetacore/zetaclient/core_context"
+)
+
+type AppContext struct {
```

Comment:
Might be because it's still a draft but we will have to add documentation comment for all the new methods